README.md

```diff
@@ -30,281 +30,19 @@
     <a href="https://www.youtube.com/channel/UCjf2teVEuYVvvVC-gFZNq6w"><img src="https://img.shields.io/badge/youtube-subscribe-fc1c1c.svg?style=flat-square"></a>
 </p>
 
-# surrealdb.go
-
-The official SurrealDB SDK for Go.
-
-## Documentation
```

Comment:
Turned into `# Links`. Please see the new Links section markdown comments for more context.